src/main/java/dev/meyi/bn/utilities/EnchantedCraftingHandler.java

```diff
@@ -0,0 +1,155 @@
+package dev.meyi.bn.utilities;
+
+import dev.meyi.bn.BazaarNotifier;
+import org.apache.commons.lang3.text.WordUtils;
+
+import java.util.*;
+
+
+public class EnchantedCraftingHandler {
+
+
+    public static ArrayList<ArrayList<String>> GetBestEnchantRecipes(){
+        ArrayList<ArrayList<String>> list = new ArrayList<>();
+        Iterator<String> NormalKeys = BazaarNotifier.enchantCraftingList.getJSONObject("Normal").keys();
+        String key1 = NormalKeys.next();
+
+        for (int i = 0; i < BazaarNotifier.enchantCraftingList.getJSONObject("Normal").length(); i++) {
+            try {
+
+                String Material = BazaarNotifier.enchantCraftingList.getJSONObject("Normal").getJSONObject(key1).getString("Material");
+                double Price1 = (BazaarNotifier.bazaarDataRaw.getJSONObject(key1).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit")) - (BazaarNotifier.bazaarDataRaw.getJSONObject(Material).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit")) * 160;
+                double Price2 = BazaarNotifier.bazaarDataRaw.getJSONObject(key1).getJSONArray("buy_summary").getJSONObject(1).getDouble("pricePerUnit") - (BazaarNotifier.bazaarDataRaw.getJSONObject(Material).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit")) * 160;
+                double ProfitPerMilC = 1000000 / (BazaarNotifier.bazaarDataRaw.getJSONObject(Material).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit") * 160);
+                double ProfitPerMil = ProfitPerMilC * BazaarNotifier.bazaarDataRaw.getJSONObject(key1).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit") - (BazaarNotifier.bazaarDataRaw.getJSONObject(Material).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit") * 160);
+
+                list.add(new ArrayList<>(Arrays.asList(Double.toString(Price1), Double.toString(Price2), Double.toString(ProfitPerMil), key1)));
+                key1 = NormalKeys.next();
+            }catch (Exception ignored){}
+        }
+
+
+        Iterator<String> OtherKeys = BazaarNotifier.enchantCraftingList.getJSONObject("Other").keys();
+        for (int i = 0; i < BazaarNotifier.enchantCraftingList.getJSONObject("Other").length(); i++) {
+
+
+            String ItemName = OtherKeys.next();
+            double ItemSellPrice = BazaarNotifier.bazaarDataRaw.getJSONObject(ItemName).getJSONArray("sell_summary").getJSONObject(1).getDouble("pricePerUnit");
+            double ItemBuyPrice = BazaarNotifier.bazaarDataRaw.getJSONObject(ItemName).getJSONArray("buy_summary").getJSONObject(1).getDouble("pricePerUnit");
+            double IngredientP= 0d;
+            int IngredientC;
+            double P= 0d;

```

Comment:
What do these represent? Can you rename them to be a bit more clear (instead of P and C)